dcap-ql/src/quote.rs

```diff
@@ -379,4 +579,52 @@ mod tests {
         assert_eq!(cd.pcesvn, EXPECTED_PCESVN);
         assert_eq!(cd.pceid, EXPECTED_PCEID);
     }
+
+    pub struct MyVerifier{}
+    impl Quote3SignatureEcdsaP256Verifier for MyVerifier {
+        fn verify_certification_data<'a>(&mut self, quote3signature: &'a Quote3SignatureEcdsaP256) -> Result<Cow<'a, [u8]>> {
+           let cert_chain = quote3signature.certification_data::<Qe3CertDataPckCertChain>()?;
+            Ok(cert_chain.certs[0].clone())
+        }
+
+        /// Verify the quoting enclave (QE3).
+        fn verify_qe3(&mut self, _qe3_report: &[u8], _authentication_data: &[u8]) -> Result<()> {
+            Ok(())
```

Comment:
How easy would it be to check this against a qe3_identity JSON? Alternatively, add a comment that this should under no circumstances be used in real code (test implementations have a habit of being copied to production)